tools/cldr-code/src/main/java/org/unicode/cldr/util/AnnotationUtil.java

```diff
@@ -0,0 +1,23 @@
+package org.unicode.cldr.util;
+
+import java.util.regex.Pattern;
+
+public class AnnotationUtil {
```

Comment:
OK, but I would just fold this into org.unicode.cldr.util.Annotations itself.

Reply:
could be future

Reply:
i considered it and then worried about initialization problems, seeing the complexity of Annotations.java and not being very familiar with it -- there's been enough excitement recently :-)